### PR TITLE
fix: focus form field enum value on create

### LINKED ISSUE #589: New Value Not Focused for Form Field Type Enum
__Describe the Bug__

When creating a new enum value for a form field the value isn't focused. Instead the value that was created first is focused.

ALROZySspI

__Steps to Reproduce__

1. Create user task
2. Create generated form field
3. Set type to enum
4. Add first value
5. _ID_ input focused
6. Add second value
7. _ID_ input of first value focused

__Expected Behavior__

_ID_ input of newly added value focused

__Environment__

- Library version: v1.x.x

### PR DESCRIPTION
**Pull request by Skaiir**
Closes #589

electron_T2Yl7jVbKt

### CODE REVIEW

File: src/provider/camunda-platform/properties/FormField.js [hunk 1]
```diff
@@ -361,7 +361,7 @@ function ValueList(props) {
```
Comment (nikku) [inline src/provider/camunda-platform/properties/FormField.js:361]: Please add `Closes #589` to the commit message.

This establishes the relevant context needed to track back the fix to the bug.

**Review by nikku — CHANGES_REQUESTED** [review]
What I'm still missing is scroll to focused element. Without that the change is often times meaningless:

capture FyL9tz_optimized

**Comment by Skaiir** [comment]
@nikku I agree with the focus but then that leads to the issue of not being able to pre-create a bunch of fields. 

If you already know you want 6 fields and click the add button a bunch of times, the + button will scroll out of view. But then again, that's already a problem for keyboard users. 

Are we fine to consider everything besides the commit amend another issue / pr? 

The missing scrollIntoView behavior IMO would need to be discussed a bit more. It would make more sense to implement it after https://github.com/camunda/camunda-modeler-design/issues/118, since this will make the + button problem irrelevant. I can set up a ticket for this if that makes sense.

**Comment by nikku** [comment]
> @nikku I agree with the focus but then that leads to the issue of not being able to pre-create a bunch of fields.

Understood. There is no clear "winner" for UX. On the other hand sections will [some day be sticky](https://github.com/camunda/camunda-modeler-design/issues/118), so important controls won't hopefully vanish too easily from your sight.

Please create another issue for the focus case. I'd love to discuss it with @andreasgeier; maybe we can even ignore it for now.

**Comment by nikku** [comment]
If you just add the `Closes #asd` comment and we have the follow up issue this one looks ready to merge for me.

**Comment by Skaiir** [comment]
> If you just add the `Closes #asd` comment and we have the follow up issue this one looks ready to merge for me.

Done, for some reason I thought I already did this 😕

**Review by nikku — APPROVED** [review]
Works for me.

**Pull request merged**